lib/taurus/qt/qtgui/taurusgui/taurusgui.py

```diff
@@ -25,7 +25,7 @@
 
 """This package provides the TaurusGui class"""
 
-__all__ = ["DockWidgetPanel", "TaurusGui"]
+__all__ = ["DockWidgetPanel", "TaurusGui", "main"]
```

Comment:
> Which generated launcher?

It is done  for the gui launcher (for the method called in the entry point).
I did in the run method : 'from taurus.qt.qtgui.taurusgui import main' 

Checking  taurus 3 this can be done importing the main like this:
 `from taurus.qt.qtgui.taurusgui.taurusgui import main` 

I will change the code.